api/routes/ACTION_router.js

```diff
@@ -0,0 +1,88 @@
+// EXPRESS & ROUTER
+const router = require("express").Router();
+
+// IMPORT MODEL
+const actionModel = require("../../data/helpers/actionModel.js");
+
+// GET
+router.get("/", (req, res) => {
+  console.log("allActions Router GET/");
+
+  actionModel
+    .get()
+    .then(allActions => {
+      console.log("allProjects", allActions);
+      allActions && res.status(200).json(allActions);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
+// GET ID
+router.get("/:id", (req, res) => {
+  console.log("actionModel Router GET id");
+  const { id } = req.params;
+
+  actionModel
+    .get(id)
+    .then(result => {
+      console.log(`GET BY ID`, result);
+      res.status(200).json(result);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
+// POST
+router.post("/", (req, res) => {
+  console.log("actionModel post/");
+  const newAction = req.body;
+  console.log("actionModel", newAction);
+  // -- //
+  actionModel
+    .insert(newAction)
+    .then(result => {
+      console.log("Insert Project", result);
+      res.status(201).json(result);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
+// PUT
+router.put("/:id", (req, res) => {
+  console.log("actionModel Put/");
+  const { id } = req.params;
+  console.log(id);
+  const updateAction = req.body;
```

Comment:
i usually dont like passing dot notation directly into a function. at the very least you should be console logging what this is and even better to save it into a variable, check that variable, and then pass it on to the function